en/dev_setup/generic_dev_tools.md

```diff
@@ -6,13 +6,24 @@ This section lists useful additional development tools that are not part of the
 
 Download and install the [QGroundControl Daily Build](https://docs.qgroundcontrol.com/en/releases/daily_builds.html).
 
+:::tip
+The *daily build* includes development tools that hidden in release builds. 
+It may also provide access to new PX4 features that are not yet supported in release builds.
+:::
+
 ![QGroundControl](../../assets/toolchain/qgc_goto.jpg)
 
-## Editor / IDE
 
-The development team often use:
+## IDE / Code Editors
+
+The development team recommend [Visual Studio Code (VSCode)](../dev_setup/vscode.md), a popular open source IDE.
+VSCode is the "officially supported" IDE, primarily because it is the easiest of the IDEs to set up and use with PX4.
 
-* [Visual Studio Code](../dev_setup/vscode.md): Popular new open source IDE
+The following IDEs also work (but are not recommended unless you're an expert):
 * [Eclipse for C/C++](https://www.eclipse.org/downloads/eclipse-packages/): Feature rich Java based IDE
 * [Sublime Text](https://www.sublimetext.com): Fast and lean text editor. 
-* [Qt Creator](https://www.qt.io/download-open-source): Popular cross-platform open-source IDE.
+* [Qt Creator IDE](../dev_setup/qtcreator.md): Popular cross-platform open-source IDE.
+
+:::note
+There is an [Eclipse project](https://github.com/PX4/PX4-Autopilot/blob/master/eclipse.project) and a [Sublime project](https://github.com/PX4/PX4-Autopilot/blob/master/Firmware.sublime-project) in the source tree.
```

Comment:
Would it be worth noting there's also a VS code config? It just gets used automatically when opening that folder.
https://github.com/PX4/PX4-Autopilot/tree/master/.vscode

Reply:
I decided against this. The difference being that a user has to know about these ones, but they don't need to know about the VSCode one.